README.md

```diff
@@ -39,6 +39,7 @@ Start by making a copy of `.env.template` and renaming it `.env`. For the backen
 * **NACHET_AZURE_STORAGE_CONNECTION_STRING**: Connection string to access external storage (Azure Blob Storage).
 * **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed model for inferencing. 
 * **NACHET_MODEL_ENDPOINT_ACCESS_KEY**: Key used when consuming online endpoint.
+* **NACHET_SEED_INFO_URL**: Url to access json containing all seeds info
```

Comment:
1) I renamed the repo nachet-db to nachet-data as discussed with @JolanThomassin , those are not the same things.

2) instead of having one URL per type of data you want, I suggest instead having:

NACHET_DATA=https://github.com/ai-cfia/nachet-data/blob/main/

and in your backend code, accessing $NACHET_DATA/seeds/all.json

This will allow you to prototype a number of additional sources of information as static json files that you can swap in for function calls to nachet-db once that becomes available.